.github/workflows/ci.yaml

```diff
@@ -87,6 +87,49 @@ jobs:
         name: build-artifact-fedora-latest-${{ matrix.build_type }}
         path: builddir/meson-logs/
 
+  almalinux:
+    runs-on: ubuntu-latest
+    container:
+      image: almalinux:latest
+      options: --privileged  -v /usr/src:/usr/src -v /lib/modules:/lib/modules
+    strategy:
+      fail-fast: false
+      matrix:
+        build_type: [release, debugoptimized]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Initialize
+      run: |
+        echo "$HOME/.poetry/bin" >> $GITHUB_PATH
+        dnf install -y sudo dnf-plugins-core epel-release
+        dnf config-manager --set-enabled powertools
+        dnf install -y which git pkg-config java-1.8.0-openjdk-devel libmicrohttpd-devel userspace-rcu-devel libyaml-devel lz4-devel libbsd-devel libcurl-devel
+        dnf install -y xz mongo-c-driver libbson openssl-devel cyrus-sasl-devel ncurses-devel cmake make automake gcc gcc-c++ kernel-devel
+        dnf install -y python36 python3-devel libxml2 libxslt
+        curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3
```

Comment:
Couldn't get it to work with pip? Doesn't matter to me either way.

Reply:
It would be nice to remove any usages of `which` and replace with the posix compliant `command -v` when we get a chance. One less thing to configure.

Reply:
For some reason pip install wanted me to install setuptools_rust. I think the usage for which is only for debugging, I can take that out.

Reply:
Whichever way works. setuptools_rust is used by the cryptography package in python.